python/ray/dataframe/dataframe.py

```diff
@@ -4917,9 +4917,8 @@ def __array__(self, dtype=None):
         return to_pandas(self).__array__(dtype=dtype)
 
     def __array_wrap__(self, result, context=None):
-        raise NotImplementedError(
-            "To contribute to Pandas on Ray, please visit "
-            "github.com/ray-project/ray.")
+        # TODO: This is very inefficient, see also __array__ and as_matrix
```

Comment:
Should we add a warning here?

Reply:
I don't think it would be good to add a warning. It would warning every time users run `np.power` or whatever. I am adding it to the documentation page, so hopefully that would be enough.